### PR TITLE
Add Decoder Extension - Update extensions.json

### PR DESCRIPTION
**Pull request by bitkarrot**
add decoder extension

### CODE REVIEW

**Comment by motorina0** [comment]
The functionality is useful, but should it be an extension?

**Comment by bitkarrot** [comment]
its tiny. I made it because i didn't want to bother you guys and it was convenient. If you want to roll into core, that's cool by me too. 

@motorina0 @dni

**Comment by dni** [comment]
added a pr to the extensions, after this i am fine with merging it. 

https://github.com/bitkarrot/decoder/pull/1

can you release and use a version with v, like `v0.0.2`

**Comment by bitkarrot** [comment]
@dni here is the latest with your PR merged, lmk if that is ok

https://github.com/bitkarrot/decoder/releases/tag/v0.0.5

**Comment by motorina0** [comment]
@bitkarrot just an idea, do you think this extension would be more accessible if it was served as a public page?
For example: https://demo.lnbits.com/decoder/check

Is there any reason for requesting a user login?

**Comment by bitkarrot** [comment]
Agree we should make the page accessible as a public page, similar to https://lightningdecoder.com/

where is the user login? its just an extension, there should be no login required? @motorina0 

> @bitkarrot just an idea, do you think this extension would be more accessible if it was served as a public page? For example: https://demo.lnbits.com/decoder/check
> 
> Is there any reason for requesting a user login?

**Comment by bitkarrot** [comment]
About the public page - it might be good to make this an opt in or out, as a bot could come along and just spam the input box relentlessly.

**Comment by bitkarrot** [comment]
Had a brief chat with @motorina0 

we're just gonna leave the decoder extension as-is. If there is nothing else you need fixed, please to add?

**Review by dni — APPROVED** [review]

File: extensions.json [hunk 1]
```diff
@@ -41,6 +41,16 @@
             "archive": "https://github.com/lnbits/copilot/archive/refs/tags/v0.5.1.zip",
             "hash": "2ee8e5cbea86786692dff3bf2346064292451d4cc3c1c7b44cd0f9ea9c5994dc"
         },
+        {
+            "id": "decoder",
+            "repo": "https://github.com/bitkarrot/decoder",
+            "name": "Decoder",
+            "version": "0.0.4",
```
Comment (motorina0) [inline extensions.json:48]:
```suggestion
            "version": "0.0.5",
```

**Comment by bitkarrot** [comment]
> CI failing at `check-json `

fixed check-json but jmeter is failing, also fails on lncalendar pr, unclear why 

```
1722580508975,1440,Fetch vetted extension list,404,Not Found,Thread Group 1-1,text,false,"Test failed: code expected to equal /

```

**Comment by motorina0** [comment]
> > CI failing at `check-json `

> ```

ah, its because the PR is from a different repo ([bitkarrot:patch-1](https://github.com/bitkarrot/lnbits-extensions/tree/patch-1)), but it tries to find the branch on this repo

```
https://raw.githubusercontent.com/lnbits/lnbits-extensions/patch-1/extensions.json
```

@dni can you please have a look

**Comment by motorina0** [comment]
All good except the CI fix which is not related to this PR, but to the fact that an external repo is used.

**Review by motorina0 — APPROVED** [review]

**Comment by bitkarrot** [comment]
updated branch LFG!!

**Review by dni — APPROVED** [review]

**Review by talvasconcelos — APPROVED** [review]
I'd bet we had `decode` endpoint in the core API...

**Comment by bitkarrot** [comment]
Whooops i just squashed and merged, i thought it was just merge :O

**Comment by motorina0** [comment]
> Whooops i just squashed and merged, i thought it was just merge :O

All good! Congrats on the extension!

**Pull request merged**